crates/exex/src/atlantic/endpoints.rs

```diff
@@ -0,0 +1,228 @@
+use super::error::SharpSdkError;
+use url::Url;
+
+/// The L2 endpoints.
+#[derive(Debug, Clone)]
+pub struct L2Endpoints {
+    /// Submit an Atlantic proof generation query
+    pub proof_generation: Url,
+    /// Submit an Atlantic trace generation query
+    pub trace_generation: Url,
+    /// Submit an Atlantic trace generation and proof generation query
+    pub trace_generation_proof_generation: Url,
+    /// Submit a L2 Atlantic query
+    pub atlantic_query: Url,
+    /// Submit a L2 Atlantic proof generation and proof verification query
+    pub proof_generation_verification: Url,
+    /// Submit a L2 Atlantic proof verification query
+    pub proof_verification: Url,
```

Comment:
not all these are "l2" as per the doc

Reply:
I've seen the same, but as per

https://github.com/chudkowsky/herodotus_sharp_playground/blob/ed6a31b34f854fefacee4312fba6dff4d57bfe6d/src/models.rs#L102-L110

everything is in L2Endpoints. Since this guy has already run jobs on it, I guess he knows what it's about. Also, the documentation is currently not complete on herodotus so in reality we don't really know precisely what each endpoint does.

I also had to rename some endpoints that didn't seem to me to be in accordance with the docs compared to the client. Since we're not sure, I'll leave it here for now as per the example client we have and we can change it in the future.

Reply:
if some things are unclear, we should ask, but **never** copy code just because we don't know